versioning-and-branching.md

```diff
@@ -60,6 +60,8 @@ git remote add upstream git@github.com:usnistgov/OSCAL.git
 
 ## Personal Working Branches
 
+Staff on the NIST OSCAL Team can do work on branches in this repository or choose to do work as an individual outside contributor in a fork as described below.
```

Comment:
Suggestion: "NIST OSCAL Team members may complete their work on branches in this repository or choose to do work in a personal fork, as an individual community contributor, as described below."

Reply:
I would prefer not to make that change because it implies that when they do so, they become a community contributor and not an employee doing this (on work time even). This is on me and my choice of preposition. I will change the preposition and modify some wording.